### PR TITLE
TEST/APPS: add verbose to test_ucx_tls.py

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
add verbose to test_ucx_tls.py

## Why ?
for debugging purpose

### CODE REVIEW

**Comment by evgeny-leksikov** [comment]
bot:pipe:retest

**Comment by evgeny-leksikov** [comment]
bot:pipe:retest

**Comment by evgeny-leksikov** [comment]
bot:pipe:retest

File: test/apps/test_ucx_tls.py [hunk 1]
```diff
@@ -79,6 +80,13 @@
     "mlx5_override"   : mlx5_am_override
 }
 
+def exec_cmd(cmd):
+    if options.verbose:
+        print cmd
+
+    return commands.getstatusoutput(cmd)
```
Comment (yosefe) [inline test/apps/test_ucx_tls.py:87]: maybe print the output as well?

File: test/apps/test_ucx_tls.py [hunk 2]
```diff
@@ -118,36 +126,47 @@ def test_fallback_from_rc(dev, neps) :
...
+                  help = "verbose output", default=False)
+(options, args) = parser.parse_args()
+
+if options.prefix == None:
     bin_prefix = "./src/tools/info"
+else:
+    bin_prefix = options.prefix + "/bin"
+
+if not (os.path.isdir(bin_prefix)):
+    parser.print_help()
```
Comment (yosefe) [inline test/apps/test_ucx_tls.py:151]: print that dir %s does not exist

**Pull request merged**